src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/SamlEntityType.php

```diff
@@ -99,14 +100,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         ]
                     )
                     ->add(
-                        'acsLocation',
-                        TextType::class,
+                        'acsLocations',
+                        CollectionType::class,
                         [
+                            'error_bubbling' => false,
+                            'prototype' => true,
+                            'allow_add' => true,
+                            'allow_delete' => true,
                             'required' => false,
+                            'entry_type' => TextType::class,
+                            'entry_options' => [
+                                'attr' => [
+                                    'data-parsley-redirecturis_set' => 'true',
+                                    'data-parsley-redirecturis_valid' => 'true',
+                                    'data-parsley-urlstrict' => null,
+                                ],
```

Comment:
Stricktly speaking these parsley validations are for the redirect urls. In an ideal world we would create a set of validations for the acs locations. A prime example of where DRY is not always the rule to follow.